hera_mc/tests/test_correlator.py

```diff
@@ -1484,79 +1482,35 @@ def test_get_node_snap_from_serial_multiple_locs(mcsession):
     connection.start_gpstime = 1230375618
     mcsession.add(connection)
     mcsession.commit()
-    node, snap_loc_num = checkWarnings(
-        mcsession._get_node_snap_from_serial, ['SNPD000703'],
-        message='Multiple snap location numbers returned')
-    assert node == 701
-    assert snap_loc_num is None
+    pytest.raises(ValueError, mcsession._get_node_snap_from_serial, ['SNPD000703'])
 
 
-def test_get_node_snap_from_serial_multiple_nodes(mcsession):
-    """Test multiple node numbers."""
-    connection = cm_partconnect.Connections()
-    connection.upstream_part = 'SNPD000703'
-    connection.up_part_rev = 'A'
-    connection.downstream_part = 'N700'
-    connection.down_part_rev = 'A'
-    connection.upstream_output_port = 'rack'
-    connection.downstream_input_port = 'loc3'
-    connection.start_gpstime = 1230375618
-    mcsession.add(connection)
-    mcsession.commit()
-    node, snap_loc_num = checkWarnings(
-        mcsession._get_node_snap_from_serial, ['SNPD000703'],
-        message='Multiple node connections returned')
-    assert node is None
-    assert snap_loc_num is None
-
-
-def test_get_node_snap_from_serial_multiple_noderevs(mcsession):
-    """Test multiple node revisions."""
-    # add part for new revision
+def test_get_node_snap_from_serial_multiple_locs(mcsession):
+    """Test multiple snap location numbers."""
     part = cm_partconnect.Parts()
-    part.hpn = 'N701'
+    part.hpn = 'SNPD000703'
     part.hpn_rev = 'B'
-    part.hptype = 'node'
-    part.manufacture_number = 'N701'
+    part.hptype = 'snap'
+    part.manufacture_number = 'D000703'
     part.start_gpstime = 1230375618
     mcsession.add(part)
     mcsession.commit()
-
     connection = cm_partconnect.Connections()
     connection.upstream_part = 'SNPD000703'
-    connection.up_part_rev = 'A'
+    connection.up_part_rev = 'B'
     connection.downstream_part = 'N701'
-    connection.down_part_rev = 'B'
+    connection.down_part_rev = 'A'
     connection.upstream_output_port = 'rack'
-    connection.downstream_input_port = 'loc3'
-    connection.start_gpstime = 1230375718
+    connection.downstream_input_port = 'loc2'
+    connection.start_gpstime = 1230375618
     mcsession.add(connection)
     mcsession.commit()
-    node, snap_loc_num = checkWarnings(
-        mcsession._get_node_snap_from_serial, ['SNPD000703'],
-        message='Multiple node connections returned')
+    node, snap_loc_num = checkWarnings(mcsession._get_node_snap_from_serial, ['SNPD000703'])
```

Comment:
You need to specify the warning message you expect -- that's the whole point of checkWarnings.